### PR TITLE
Bookmark enhancement

### PR DESCRIPTION
**Pull request by SarahhhC**
* Refresh issue solved.
* Bookmark list update issue solved.
* Image changed.

### CODE REVIEW

File: src/components/Bookmark/Bookmark.js [hunk 1]
```diff
@@ -38,6 +38,12 @@ class Bookmark extends Component {
   }
```
Comment (MechanicKim) [inline src/components/Bookmark/Bookmark.js:36]: line 5, 9
'Dimensions', 'ScrollView' are defined but never used

File: src/components/Bookmark/Bookmark.js [hunk 2]
```diff
@@ -38,6 +38,12 @@ class Bookmark extends Component {
...
```
Comment (MechanicKim) [inline src/components/Bookmark/Bookmark.js]: Trailing spaces not allowed
line 21, 31, 46, 55, 59, 64, 75, 79, 102, 107 have same error.

**Comment by MechanicKim** [comment]
I reviewed this PR.
There are more style errors except the lines on PR.
But I didn't comment.

File: src/components/Bookmark/Bookmark.js (hunk 1 above)
Comment (SarahhhC) [reply src/components/Bookmark/Bookmark.js:36]: True, thanks!

File: src/components/AppProps.js [hunk 3]
```diff
@@ -280,11 +280,17 @@ const AppProps = {
       rightTitle: 'right',
       rightButtonImage: bookmarkButton,
       rightButtonIconStyle: styles.rightBtnBookmark,
-      onRight: () => Alert.alert('bookmark'),
+      onRight: () => {},
```
Comment (ifndefdeadmau5) [inline src/components/AppProps.js:283]: This function does nothing can we delete?

**Review by ifndefdeadmau5 — APPROVED** [review]
Looks great. LGTM 👍

File: src/components/AppProps.js (hunk 3 above)
Comment (SarahhhC) [reply src/components/AppProps.js:283]: It's for defining in advance!

**Pull request merged**